library/time-library/pom.xml

```diff
@@ -26,6 +26,11 @@
 
     <dependencies>
         <dependency>
+            <groupId>org.roaringbitmap</groupId>
```

Comment:
Do  you need to add this dependency, it should be included in the bitmap-library dependency.